toolkit/scripts/imggen.mk

```diff
@@ -5,8 +5,7 @@ $(call create_folder,$(IMAGEGEN_DIR))
 
 # Resources
 config_name              = $(notdir $(CONFIG_FILE:%.json=%))
-CONFIG_BASE_DIR          = $(dir $(CONFIG_FILE:%.json=%))
-config_other_files       = $(if $(CONFIG_FILE),$(shell $(SCRIPTS_DIR)/get_config_deps.sh $(CONFIG_FILE) $(CONFIG_BASE_DIR)),)
+config_other_files       = $(if $(CONFIG_FILE),$(shell $(SCRIPTS_DIR)/get_config_deps.sh $(CONFIG_FILE)),)
```

Comment:
```suggestion
config_other_files       = $(if $(CONFIG_FILE),$(call shell_real_build_only, $(SCRIPTS_DIR)/get_config_deps.sh $(CONFIG_FILE)),)
```

Reply:
This function will only execute if its a "real" build, I don't think we care about any of these outputs when its a dry-run (ie tab completing).